lib/workers/repository/update/branch/index.spec.ts

```diff
@@ -2015,6 +2015,27 @@ describe('workers/repository/update/branch/index', () => {
       expect(commit.commitFilesToBranch).toHaveBeenCalled();
     });
 
+    it('continues when checked by checkedBranches', async () => {
+      getUpdated.getUpdatedPackageFiles.mockResolvedValueOnce({
+        ...updatedPackageFiles,
+      });
```

Comment:
```suggestion
      getUpdated.getUpdatedPackageFiles.mockResolvedValueOnce(updatedPackageFiles);
```
i think this should be simpler